src/items/itemsTable.component.tsx

```diff
@@ -53,8 +60,13 @@ export function ItemsTable(props: ItemTableProps) {
     return [
       {
         header: 'ID',
-        accessorFn: (row) => row.id,
         size: 250,
+        accessorFn: (row) => row.id,
```

Comment:
As a note - although I think it only effects the table caching issue I saw https://www.material-react-table.com/docs/guides/data-columns#method-2---using-an-accessorfn-and-id says an `id` is required when using `accessorFn`, as it works perhaps its worth just making a note/an issue to add them in as a later non-mvp maintenance task?

Reply:
#250